src/components/RoomView/gameLog.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import './roomView.css';
+import GuessesRemaining from './GuessesRemaining';
+
+interface ClueType {
+clueString: string;
+clueNumber: number;
+}
+
+interface GameState {
+gameHistory: (string | ClueType)[];
+status: string;
+}
+
+const GameLog: React.FC = () => {
+const game = useSelector<{ game: GameState }, GameState>(
+(state) => state.game
+);
+const gameHistory = game.gameHistory;
+const gameStatus = game.status;
+
+return (
+<div className='gameLog'>
```

Comment:
Looking at the spacing here, I wonder if youre prettier isn't working?

Reply:
it is working, I think I manually entered that line so it is easier for me to look

Reply:
Github definitely displaying this strangely then, I see when i look at this branch in vscode it has more spacing/indentation!